<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsLoc</class>
 <widget class="QWidget" name="QtWidgetsLoc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件加载</string>
  </property>
  <widget class="QLabel" name="labelWind">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>地点文件加载</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLoc">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>地点文件</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditLoc">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLoc">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDone">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQuit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHits">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>561</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonDone</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsLoc</receiver>
   <slot>checkFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonLoc</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsLoc</receiver>
   <slot>inputLoc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonQuit</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsLoc</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>466</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>inputLoc()</slot>
  <slot>checkFile()</slot>
 </slots>
</ui>
